src/nft/components/details/AssetPriceDetails.tsx

```diff
@@ -140,11 +146,51 @@ const OwnerText = styled.a`
 const OwnerInformationContainer = styled.div`
   color: ${({ theme }) => theme.textSecondary};
   display: flex;
+  flex-direction: column;
   justify-content: space-between;
   padding: 0 8px;
   margin-bottom: 20px;
 `
 
+const AssetInfoContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 8px;
+`
+
+const AssetHeader = styled.div`
+  display: -webkit-box;
+  align-items: center;
+  font-size: 28px;
+  font-weight: 500;
+  line-height: 36px;
```

Comment:
no to the above - we need to add a lot to themedText

Reply:
Mind elaborating a bit on that? While it clearly lacks some features at the moment, I think it's a great addition that is being rolled out gradually in a lot of incoming PRs. 

If you feel like it's lacking some features, I think it would be great to define them and resolve (maybe even before PRs like this are merged), so that we don't make ourselves more additional work in the future.

Reply:
In this particular example, I am not sure what's stopping us from wrapping a ThemedText in a `styled.div`. You can always override `line-height`, if that's the intent.

Reply:
Looks like `CollectionHeader` could also wrap a `ThemedText`, font weight 400, font-size 16 and textPrimary is exactly what BodyPrimary defines.